server/src/seeders/20190315112223-demo-user.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-unused-vars */
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.bulkInsert('users', [
+    {
+      id: '7139d3af-b8b4-44f6-a49f-9305791700f4',
+      firstName: 'amaechi',
+      lastName: 'chuks',
+      email: 'amaechichuks2000@yahoo.com',
+      password: 'chuks',
```

Comment:
I don’t think putting the password here is best practice. We can make use of .env and bcrypt.

Reply:
Noted